app/views/content_items/_attachments.html.erb

```diff
@@ -0,0 +1,14 @@
+<% attachment_title_id = "#{title.parameterize}-title" %>
+
+<% if legacy_pre_rendered_documents.present? %>
+  <%= render 'govuk_publishing_components/components/heading',
+      id: attachment_title_id,
+      text: title,
+      mobile_top_margin: true %>
+
+  <div aria-labelledby="<%= attachment_title_id %>">
```

Comment:
Wouldn't this have the effect of reading the heading out twice (once on the heading itself, and once immediately afterwards)? (NB, the examples I've seen on MDN tend to show the heading inside the div):

```erb
<div aria-labelledby="<%= attachment_title_id %>">

  <%= render 'govuk_publishing_components/components/heading',
      id: attachment_title_id,
      text: title,
      mobile_top_margin: true %>
```

I'm sure Alex has already looked at this, I'm just not immediately clear on how this is an improvement 🤔

Reply:
Looks like this is a case of me assuming we were doing a good thing in the first place 🤦‍♂. This is replicating a behaviour that [previously applied only to publication documents](https://github.com/alphagov/government-frontend/pull/1709/commits/6287f6847acf2fb6ef9024fd3fce5042186bedb8#diff-24095b2b1c8348342f68a199663a50b7R37), and applying it to all lists of attachments (with a title).

Apparently it was introduced [here](https://github.com/alphagov/government-frontend/pull/256/commits/da1bc39eaf8b3839af79ea5f4f4adfa910f60203), but with no explanation. I've added another commit so it conforms [to the spec](https://www.w3.org/TR/wai-aria/#region). Removing it makes the tests harder, as there's nothing to 'grab hold of', so I've gone with fixing it, instead.

How does it look now?